CONTRIBUTING.md

```diff
@@ -52,3 +52,11 @@ To do this, run:
 ```shell
 npm run start
 ```
+
+### Releasing a new version
+
```

Comment:
Could you specify here that someone will need to have access to the npm organisation? Just to avoid confusion.